spec/features/form/add_or_edit_questions/add_a_question_for_each_type_of_answer_spec.rb

```diff
@@ -35,6 +35,45 @@
         and_i_add_another_question
       end
     end
+
+    context "when the form is in a group which has long lists enabled" do
+      let(:group) { create(:group, organisation: standard_user.organisation, long_lists_enabled: true) }
+      let(:options) { 10.times.map { Faker::Name.unique.name } }
+
+      scenario "add a selection question which the user can answer more than once" do
+        when_i_am_viewing_an_existing_form
+        and_i_want_to_create_or_edit_a_page
+        and_i_select_selection_type_question
+        and_i_select_people_can_choose_one_or_more_options
+        and_i_configure_the_selection_options(options)
+        and_i_click_save
+        and_i_add_another_question
+      end
+
+      scenario "add a selection question which the user can answer once" do
+        when_i_am_viewing_an_existing_form
+        and_i_want_to_create_or_edit_a_page
+        and_i_select_selection_type_question
+        and_i_select_people_can_only_choose_one_option
+        and_i_configure_the_selection_options(options)
+        and_i_click_save
+        and_i_add_another_question
+      end
+
+      context "when there are more than 30 options" do
```

Comment:
I wonder whether its necessary to have a feature test for this? Feature tests are slow to run, so should only cover the minimum happy path to ensure everything works together. I think we have enough unit tests to cover that we can add more than 30 options, unless I'm missing something else that this tests that isn't covered by the other 2.

Reply:
Yeah I debated this and erred on the side of more tests. But you're right that this is covered elsewhere so I'll get rid!

Reply:
Done